test/src/tests-store.ts

```diff
@@ -378,6 +383,190 @@ describe('Store', () => {
+        });
+    });
+
+    describe('#setFirstSeenTime', () => {
+        it('should set the firstSeenTime in the store', () => {
+            const store: IStore = new Store(
+                sampleConfig,
+                window.mParticle.getInstance()
+            );
+
+            store.setFirstSeenTime(testMPID, 12345);
+            expect(store.persistenceData[testMPID].fst).to.equal(12345);
+        });
+
+        it('should return undefined if mpid is null', () => {
+            const store: IStore = new Store(
+                sampleConfig,
+                window.mParticle.getInstance()
+            );
+
+            expect(store.setFirstSeenTime(null, 12345)).to.equal(undefined);
+        });
+
+        it('should set the firstSeenTime in persistence', () => {
+            const store: IStore = new Store(
+                sampleConfig,
+                window.mParticle.getInstance()
+            );
+
+            store.setFirstSeenTime(testMPID, 12345);
+            const fromPersistence = window.mParticle
+                .getInstance()
+                ._Persistence.getPersistence();
+
+            expect(fromPersistence[testMPID]).to.be.ok;
+            expect(fromPersistence[testMPID].fst).to.be.ok;
+            expect(fromPersistence[testMPID].fst).to.equal(12345);
+        });
+
+        it('should override persistence with store values', () => {
+            const persistenceValue = JSON.stringify({
+                testMPID: {
+                    fst: 12345,
+                },
+            });
+
+            localStorage.setItem(workspaceCookieName, persistenceValue);
+
+            const store: IStore = new Store(
+                sampleConfig,
+                window.mParticle.getInstance()
+            );
+
+            store.setFirstSeenTime(testMPID, 54321);
+            const fromPersistence = window.mParticle
+                .getInstance()
+                ._Persistence.getPersistence();
+
+            expect(fromPersistence[testMPID]).to.be.ok;
+            expect(fromPersistence[testMPID].fst).to.be.ok;
+            expect(fromPersistence[testMPID].fst).to.equal(54321);
+        });
+    });
+
+    describe('#getLastSeenTime', () => {
+        it('should return the lastSeenTime from the store', () => {
+            const store: IStore = new Store(
+                sampleConfig,
+                window.mParticle.getInstance()
+            );
+
+            store.persistenceData[testMPID] = {
+                lst: 12345,
+            };
+
+            expect(store.getLastSeenTime(testMPID)).to.equal(12345);
+        });
+
+        it('should return null if no lastSeenTime is found', () => {
+            const store: IStore = new Store(
+                sampleConfig,
+                window.mParticle.getInstance()
+            );
+
+            expect(store.getLastSeenTime(testMPID)).to.equal(null);
+        });
+
+        it('should return null if mpid is null', () => {
```

Comment:
to be consistent with tests, can this test be added to `getFIrstSeenTime` too?

Reply:
Good call out